holland/core/cmdshell.py

```diff
@@ -21,6 +21,11 @@
 
 LOGGER = logging.getLogger(__name__)
 
+holland_conf = '/etc/holland/holland.conf'
+currplatform = sys.platform
+if currplatform.startswith('freebsd'):
+    holland_conf = '/usr/local' + holland_conf
```

Comment:
os.path.join() cannot be used here as it will not join absolute paths. Since holland_conf is an absolute path this would fail unless it was adjusted that will then require and extra /  be added later.